### ISSUE TITLE
GPU support (plans/roadmap)

### ISSUE THREAD

**Issue by asmith26** [modular/modular#3567]
Hi there,

I'm thinking about buying a new GPU soon for neural network training/fine-tuning, and was wondering if mojo has plans to support (or currently supports) the various options:
- NVIDIA (including e.g. any particular architectures/generations, CUDA versions/CUDA capability)
- AMD (including e.g. any particular architectures, ROCm versions)
- Intel (including e.g. any particular architectures/generations, oneAPI)

Many thanks for any help! :)

**Comment by ematejska** [comment]
Yes, exactly!  Check this out for more info: [https://docs.modular.com/max/faq#what-hardware-is-currently-supported-by-max-engine](https://docs.modular.com/max/faq#what-hardware-is-currently-supported-by-max-engine).

**Comment by asmith26** [comment]
Thanks that's really helpful. Are there plans to support AMD and Intel GPUs at some point? Thanks again

**Comment by scottamain** [comment]
Sorry for the long silence on this issue! We've come a long way and you can now see all our GPU compatibility listed here: [https://docs.modular.com/max/faq#gpu-requirements](https://docs.modular.com/max/faq#gpu-requirements)
